src/main/java/io/appium/java_client/remote/options/W3CCapabilityKeys.java

```diff
@@ -23,7 +23,7 @@
 public class W3CCapabilityKeys implements Predicate<String> {
     public static final W3CCapabilityKeys INSTANCE = new W3CCapabilityKeys();
     private static final Predicate<String> ACCEPTED_W3C_PATTERNS = Stream.of(
-                    "^[\\w-]+:.*$",
+                    "^[^:]+:.*$",
```

Comment:
```suggestion
                    "^[\\w-.]+:.*$"
```
what about just adding `.` to the acceptable characters? I'm not sure it's worth allowing all characters except the colon.

Reply:
A simple `.` would be enough for me 1e5a0837b4e8e0929d9aaaaaf369a68af5781b31